.changeset/chilled-numbers-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/design-tokens': minor
+---
+
+Updated the body background color to white and darkened the overlay color.
```

Comment:
Should we also mention the new borderRadius?

Reply:
Ah it's not there, do we want to add it? Then I can make a PR to `next` that replaces all hardcoded values of `12px` and `16px` by the tokens

Reply:
I removed the commit that added the border-radius to the design tokens since it would have conflicted with your change. Can you talk to Giao-Chung and figure out which value(s) to keep, please?

Reply:
I will! 🙂

Reply:
Oops, I already did 🤦 

<img width="291" alt="Screenshot 2021-06-04 at 11 23 34" src="https://user-images.githubusercontent.com/35560568/120779422-57a8aa80-c527-11eb-902e-6af88edb2353.png">

Do you want to do it as part of this PR or should I make a separate one? Sorry about this

Reply:
Spec:

<img width="600" alt="Screenshot 2021-06-04 at 11 31 37" src="https://user-images.githubusercontent.com/35560568/120780480-76f40780-c528-11eb-9d11-f0dfa11aeef5.png">

I think we can keep the mega/giga naming for now.

What I suggest then is that I'll make a separate PR to `next` removing the old values (6px for example) and updating any component using it. I'll double check the naming with design and write a codemod to automate this when migrating to v3.

Reply:
Let's make a separate PR against the `next` branch as you suggested 👍🏻